packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -446,6 +447,7 @@ export async function createMember(parent: api.IZoweDatasetTreeNode, datasetProv
  */
 export async function openPS(
     node: api.IZoweDatasetTreeNode,
+    forceDownload: boolean,
     previewMember: boolean,
     datasetProvider?: api.IZoweTree<api.IZoweDatasetTreeNode>
```

Comment:
Thoughts on moving these 3 parameters into an `options` object?

```suggestion
    options?: Partial<OpenPSOpts>
```

And `OpenPSOpts` could be something like:
```ts
type OpenPSOpts = {
    forceDownload: boolean;
    previewMember: boolean;
    datasetProvider: api.IZoweTree<api.IZoweDatasetTreeNode>
};
```